verifier/__main__.py

```diff
@@ -0,0 +1,55 @@
+import requests
```

Comment:
WRONG PREFIX in commit message

ALL WRONG in commit message

ALL WRONG in commit message

ALL WRONG in commit message

ALL WRONG in commit message